pulp_container/app/models.py

```diff
@@ -309,6 +312,14 @@ def get_repository_version(self):
         else:
             return None
 
+    def redirect_to_content_app(self, url):
+        """
+        Add preauthentication query string to redirect attempt.
+        """
+        if self.content_guard:
+            url = self.content_guard.cast().sign_url(url)
+        return HttpResponseRedirect(url)
```

Comment:
Use `redirect()` instead of `HttpResponseRedirect`. It does the same thing and it looks a bit cleaner, see https://docs.djangoproject.com/en/3.0/topics/http/shortcuts/#redirect.